CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+4.5.0 (August 2022)
+-------------------
+
+- Support more dynamic label options (Thanks to Tom Boissonnet) ([#472](https://github.com/ome/omero-figure/pull/472))
```

Comment:
Once the section has been updated in the relevant PR, thoughts about adding a reference to https://github.com/ome/omero-figure/blob/master/docs/figure_file_format.rst#version-history in the release notes for anyone interested in more technical information on the underlying JSON model changes?

Reply:
Done in [bcf9e72](https://github.com/ome/omero-figure/pull/475/commits/bcf9e72db227430e0e93db4dbcdafc24d6f97813)